tools/velvet/velvetg_wrapper.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+"""
+Classes encapsulating decypher tool.
+James E Johnson - University of Minnesota
+"""
+import os
```

Comment:
probably, but that's a lot of work, and outside the scope of what I can reasonably justify right now :(